pkgs/applications/networking/instant-messengers/mattermost-desktop/mime-type.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/assets/linux/create_desktop_file.sh b/src/assets/linux/create_desktop_file.sh
+index 63aabe8b..dd9cf292 100755
+--- a/create_desktop_file.sh
++++ b/create_desktop_file.sh
+@@ -12,6 +12,7 @@ Comment=Mattermost Desktop application for Linux
+ Exec="${FULL_PATH}/mattermost-desktop" %U
+ Terminal=false
+ Type=Application
++MimeType=x-scheme-handler/mattermost-dev
```

Comment:
I think perhaps instead of this we should add normal mattermost handler and make sure we build with NODE_ENV=production as per comments in https://github.com/mattermost/desktop/issues/2942?

Reply:
Setting the `NODE_ENV` won't actually change that, because the `electron-is-dev` package uses `electron.app.isPackaged`. It provides an escape hatch for using an environment vairable to disable it, however, so the derivation could just add `--set-default ELECTRON_IS_DEV 0` to the `makeWrapper` flags instead of doing all of this patching

Reply:
This works @lilyinstarlight , we just tried it out :)